Sources/Testing/Support/Locked.swift

```diff
@@ -149,6 +151,34 @@ struct Locked<T>: RawRepresentable, Sendable where T: Sendable {
       }
     }
   }
+#elseif os(FreeBSD)
```

Comment:
Rather than duplicating this entire function, why not just change the argument type to `UnsafeMutablePointer<_Lock>`?

Reply:
`_Lock` is defined as private and therefore cannot be use here as this function is internal.

Reply:
The typealias can be made `internal` (no keyword) if needed.